java/ql/automodel/src/qlpack.yml

```diff
@@ -1,5 +1,5 @@
 name: codeql/java-automodel-queries
-version: 0.0.15-dev
+version: 0.0.16-dev
```

Comment:
just for my own benefit: this is releasing version `0.0.15` and in the process bumping  the qlpack yml (to `0.0.16-dev`) in preparation for the next release, right?

Reply:
I believe so, yes.